tests/network/localnet/conftest.py

```
@@ -148,7 +148,7 @@ def localnet_client(localnet_running_vms: tuple[BaseVirtualMachine, BaseVirtualM
 
 @pytest.fixture(scope="module")
 def nncp_localnet_on_secondary_node_nic(
-    worker_node1: Node, nodes_available_nics: dict[str, list[str]]
+    worker_node1: Node, hosts_common_available_ports: list[str]
```

Ruff (0.13.1)

Unused function argument: `worker_node1`

(ARG001)